test/inductor/test_extension_backend.py

```diff
@@ -101,6 +101,11 @@ def tearDown(self):
 
     def test_open_device_registration(self):
         torch.utils.rename_privateuse1_backend("extension_device")
+        torch._register_device_module('extension_device', self.module)
+
+        def _lazy_init():
```

Comment:
Shouldn't this be defined in the setup?
Also does this mean that this PR will break any privateuse1 backend that doesn't define this function?

Reply:
@albanD  torch._register_device_module is a runtime API. when test_extension_backend.py exits, any privateuse1 backend could define by torch._register_device_module.

Reply:
hi, alban @albanD , i also meet this problem when i use the privateuse1 to support our device and our device also support lazy init。Do you have other way to solve this problem?